localoam/index.js

```diff
@@ -0,0 +1,52 @@
+var http = require('http');
+var filed = require('filed');
+var glob = require('glob');
+var fs = require('fs');
+
+var master = {
+  'nodes': [
+    {
+      'locations': [
+        {
+          'type': 'localoam',
+          'url': 'http://localoam:4999'
+        }
+      ]
+    }
+  ]
+};
+
+function sendJSON (json, res) {
+  res.setHeader('Content-Type', 'application/json');
+  res.write(json);
+  res.end();
+}
+
+var server = http.createServer(function (req, res) {
+  var url = req.url;
+  if (url === '/list') {
+    // Return all the JSON metadata files with their last modified date
+    glob('/data/**/*.json', function (err, files) {
+      if (err) {
+        console.error(err);
```

Comment:
This needs to return, otherwise `sendJSON` will be called with `records === undefined` (as it is now).